app/assets/stylesheets/components/_popovers.scss

```diff
@@ -2,3 +2,29 @@
   font-size: 12px;
   line-height: 140%;
 }
+
+.toast {
+  position: fixed;
+  font-size: 16px;
+  padding: 15px;
+  opacity: 0.9;
+  border-radius: 3px;
+  box-shadow: 2px 2px 15px rgba(0, 0, 0, 0.1);
+  bottom: 15px;
+  right: 15px;
+
+  &:hover {
+    opacity: 1;
+    box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.15);
+  }
+
+  &.toast-error {
+    background: $red;
+    color: white;
+  }
+
+  &.toast-success {
+    background: green;
```

Comment:
small nit: could you replace this with `#2ecc71`? the default green isnt too flattering 😛